staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/conversion_test.go

```diff
@@ -32,31 +32,53 @@ func TestJSONConversion(t *testing.T) {
 	sliceJSON := apiextensions.JSON([]string{"foo", "bar", "baz"})
 
 	testCases := map[string]struct {
-		input    *apiextensions.JSON
-		expected *JSON
+		input    *apiextensions.JSONSchemaProps
+		expected *JSONSchemaProps
 	}{
+		"omitempty": {
+			input: &apiextensions.JSONSchemaProps{
+				Default: nil,
+			},
+			expected: &JSONSchemaProps{},
+		},
 		"null": {
-			input: &nullJSON,
-			expected: &JSON{
-				Raw: []byte(`null`),
+			input: &apiextensions.JSONSchemaProps{
+				Default: &nullJSON,
+			},
+			expected: &JSONSchemaProps{
+				Default: &JSON{
+					Raw: []byte(`null`),
```

Comment:
here I would expect also `nil` in Default. We possibly have to set nil in that case in the custom conversion.

Reply:
btw, better call `nullJSON` `nilJSON`. Had to double-check that it's actually nil.